pkg/inventory/sanitize_test.go

```diff
@@ -0,0 +1,29 @@
+package inventory
+
+import "testing"
+
+func TestSanitizeURL(t *testing.T) {
```

Comment:
May you check SSH URLs like `git@github.com:shipwright-io/build.git` ?

Reply:
Sure! I've added the support for the `git` scheme, please consider the latest changes.